aspnetcore/signalr/hubs.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Use hubs in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use hubs in ASP.NET Core SignalR.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 03/30/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/hubs
+---
+
+# Use hubs in SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel) and [Kevin Griffin](https://twitter.com/1kevgriff)
+
+## What is a SignalR hub
+
+The SignalR Hubs API enables you to call methods on connected clients from the server. In the server code, you define methods that are called by client. In the client code, you define methods that are called from the server. SignalR takes care of everything behind the scenes that makes real-time client-to-server and server-to-client communications possible.
+
+## Configure SignalR hubs
+
+The SignalR middleware requires some services to be able to run. To configure those call `services.AddSignalR`.
```

Comment:
Replace these 2 sentences with something like the following:
The SignalR middleware requires some services, which are configured by calling `services.AddSignalR`: